docs/all_linters.md

```diff
@@ -19,7 +19,7 @@
 | [**clj-kondo**](https://github.com/borkdude/clj-kondo){target=_blank}                                   |   2021.06.18   |   <!-- -->   | [CLOJURE](descriptors/clojure_clj_kondo.md)                                                                                                                                                |       :heart:       |                [MegaLinter reference](https://github.com/borkdude/clj-kondo/blob/master/doc/ci-integration.md#github){target=_blank}                |
 | [**coffeelint**](https://github.com/clutchski/coffeelint){target=_blank}                                |     5.2.0      |    Other     | [COFFEE](descriptors/coffee_coffeelint.md)                                                                                                                                                 |   :white_circle:    |                                        [Repository](https://github.com/clutchski/coffeelint){target=_blank}                                         |
 | [**cpplint**](https://github.com/cpplint/cpplint){target=_blank}                                        |     1.5.5      |   <!-- -->   | [C](descriptors/c_cpplint.md)<br/> [CPP](descriptors/cpp_cpplint.md)                                                                                                                       |   :white_circle:    |                                            [Web Site](https://github.com/cpplint/cpplint){target=_blank}                                            |
-| [**cspell**](https://github.com/streetsidesoftware/cspell){target=_blank}                               |     5.13.1     |     MIT      | [SPELL](descriptors/spell_cspell.md)                                                                                                                                                       |       :heart:       |             [MegaLinter reference](https://github.com/streetsidesoftware/cspell/tree/master/packages/cspell#mega-linter){target=_blank}             |
+| [**cspell**](https://github.com/streetsidesoftware/cspell){target=_blank}                               |     5.13.1     |     MIT      | [SPELL](descriptors/spell_cspell.md)                                                                                                                                                       |       :heart:       |              [MegaLinter reference](https://github.com/streetsidesoftware/cspell/tree/main/packages/cspell#mega-linter){target=_blank}              |
```

Comment:
I don't understand why MegaLinter added these two spaces, but I won't question it for now.

Reply:
markdown-table-formatter ;)